handlers/handler-validator/src/main/java/org/apache/servicecomb/validator/ParamValidateHandler.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.servicecomb.validator;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.Validation;
+import javax.validation.Validator;
+
+import org.apache.servicecomb.core.Handler;
+import org.apache.servicecomb.core.Invocation;
+import org.apache.servicecomb.swagger.invocation.AsyncResponse;
+import org.apache.servicecomb.swagger.invocation.Response;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * ParamValidateHandler
+ * Handler to validate the input request parameters 
+ *
+ */
+public class ParamValidateHandler implements Handler {
+  private static final Logger LOGGER = LoggerFactory.getLogger(ParamValidateHandler.class);
+
+  private static Validator validator = Validation.buildDefaultValidatorFactory().getValidator();
+
+  @Override
+  public void handle(Invocation invocation, AsyncResponse asyncResp) throws Exception {
+    Object[] args = invocation.getArgs();
+    boolean invalid = false;
+    List<Object> errList = new ArrayList<>();
```

Comment:
both solution and code is too complex
i remember it's very simple to IMPL validator, i'll give a example later

Reply:
have you showed the example to sukesh? @wujimin